Source/WebCore/Modules/modern-media-controls/controls/media-controls.js

```diff
@@ -38,14 +38,19 @@ class MediaControls extends LayoutNode
         this.layoutTraits = layoutTraits;
 
         this.playPauseButton = new PlayPauseButton(this);
-        this.airplayButton = new AirplayButton(this);
         this.pipButton = new PiPButton(this);
         this.fullscreenButton = new FullscreenButton(this);
         this.muteButton = new MuteButton(this);
         this.tracksButton = new TracksButton(this);
-        this.overflowButton = new OverflowButton(this);
         this.volumeButton = new VolumeButton(this);
-        this.brightnessButton = new BrightnessButton(this);
+
+        if (this.layoutTraits.isApplePlatform()) {
+            this.overflowButton = new OverflowButton(this);
+            this.brightnessButton = new BrightnessButton(this);
```

Comment:
ditto (:48), these buttons by themselves dont really do anything.  they have to be added to some `children` somewhere in order to have any effect